File: client/mc/common2/src/main/java/calebxzhou/rdi/mc/common2/mcp/RMcpPlayerDetailData.java
package calebxzhou.rdi.mc.common2.mcp;

import java.util.Map;

public record RMcpPlayerDetailData(
        RMcpPlayerData brief,
        RMcpEntityDetailData entity,
        Map<String, Object> profile,
        Map<String, Object> player,
        Map<String, Object> inventory
) {
}

File: mc/rmcp/common/src/main/kotlin/calebxzhou/rdi/mc/common2/mcp/RMcpPlayerDetailData.kt
package calebxzhou.rdi.mc.common2.mcp

@JvmRecord
data class RMcpPlayerDetailData(
    val brief: RMcpPlayerData,
    val entity: RMcpEntityDetailData,
    val profile: Map<String, Any?>,
    val player: Map<String, Any?>,
    val inventory: Map<String, Any?>?
)
